src/I18nContext.spec.js

```diff
@@ -1,53 +1,15 @@
 // Core
 import React from 'react';
-import ReactDOM from 'react-dom';
-// Test
-import {act} from 'react-dom/test-utils';
+// Testing
 import test from 'ava';
+import render from '@test/customRender';
```

Comment:
I would like see this more likely the @testing-library/react, using the same api but including a new method

```javascript
import { customRender, cleanup, ...(rest of the original utilities) } from '@test/react';
````